crates/viewer/re_view_spatial/tests/transform_clamping.rs

```diff
@@ -234,14 +235,17 @@ fn run_view_ui_and_save_snapshot(
             // * NO other boxes/spheres, in particular no magenta ones!
 
             let name = format!("{name}_{target}");
-            let raw_input = harness.input_mut();
-            raw_input
-                .events
-                .push(egui::Event::PointerMoved((100.0, 100.0).into()));
-            raw_input.events.push(egui::Event::MouseWheel {
-                unit: egui::MouseWheelUnit::Line,
-                delta: egui::Vec2::UP * 2.0,
-                modifiers: egui::Modifiers::default(),
+            test_context.with_blueprint_ctx(|ctx, _| {
+                ViewProperty::from_archetype::<EyeControls3D>(
+                    ctx.current_blueprint(),
+                    ctx.blueprint_query(),
+                    view_id,
+                )
+                .save_blueprint_component(
+                    &ctx,
+                    &EyeControls3D::descriptor_position(),
+                    &Position3D::new(15.0, 5.0, 15.0),
+                );
```

Comment:
given how often this shows up it makes me wonder whether we should have a utility specificially in `re_view_spatial/tests` to set the camera :)

we can put that in a follow-up tho / whoever gets annoyed enough by this